zone/gm_commands/setanim.cpp

```diff
@@ -2,9 +2,24 @@
 
 void command_setanim(Client *c, const Seperator *sep)
 {
-	int arguments = sep->argnum;
+	const auto arguments = sep->argnum;
 	if (!arguments || !sep->IsNumber(1)) {
-		c->Message(Chat::White, "Usage: #setanim [Animation ID (IDs are 0 to 4)]");
+		c->Message(Chat::White, "Usage: #setanim [Animation ID]");
```

Comment:
We should still inform the user what the options are (0-4) in some way shape or form. Also wouldn't hurt to do input validation to make sure that animation_id falls within that range as well

Reply:
I validate below. All animation IDs and names are also sent on invalid input.

Reply:
Roger